examples/diagnostics/compare_diags.py

```diff
@@ -27,6 +27,29 @@
 # Methods
 # ======================================================================
 
+def read_config_file():
+    """
+    Reads configuration information from a YAML file.
+    """
+
+    # Take the config file as the 2nd argument (or use a default)
+    # NOTE: sys.argv[0] is always the program name!
+    if len(sys.argv) == 2:
+        config_file = sys.argv[1]
+    else:
+        config_file = "compare_diags.yml"
+
+    # Read the configuration file in YAML format
+    try:
+        print("compare.diags.py: Using configuration file " + config_file)
+        config = yaml_load_file(open(config_file))
+    except Exception:
+        msg = "compare_diags.py: Error reading configuration file: " + \
+            config_file
+        raise Exception(msg)
+
+    return config
```

Comment:
@yantosca is there a reason you decided to stop using `util.read_config_file()`? It seems like this function performs essentially the same steps? Albeit with `load` instead of `safe_load` and slightly more detailed error messages.

Reply:
@laestrada, my use of read_config_file may have pre-dated `util.read_config_file`.  I can use the one in util